enterprise/backend/src/metabase_enterprise/semantic_search/index.clj

```diff
@@ -145,18 +145,18 @@
    :text_search_vector  (if (:name doc)
                           [:||
                            (search/weighted-tsvector "A" (:name doc))
-                           (search/weighted-tsvector "B" (:searchable_text doc ""))]
-                          (search/weighted-tsvector "A" (:searchable_text doc "")))
+                           (search/weighted-tsvector "B" (or (:searchable_text doc) ""))]
+                          (search/weighted-tsvector "A" (or (:searchable_text doc) "")))
```

Comment:
Could this be replaced with the following?

```
[:||
  (search/weighted-tsvector "A" (or (:name doc) ""))
  (search/weighted-tsvector "B" (or (:searchable_text doc) ""))]
```

Reply:
Meaning not have the `if`?

Reply:
I had left them with the `if` because the A vs B controls the weighting and I don't think we want the description to be weighted differently than other descriptions just because the row doesn't have a name.

So I think I'll leave it as it is, even though it is a bit more complex